### PR TITLE
Use browserify-gem and remove gulp workaround.

### PR DESCRIPTION
**Pull request by lennerd**

### CODE REVIEW

File: app/assets/javascripts/test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const bla = 'Test';
+
+class Test {
+
+}
```
Comment (schultyy) [inline app/assets/javascripts/test.js]: As discussed, this can be removed

File: Gemfile [hunk 2]
```diff
@@ -86,6 +86,8 @@ group :default do
   gem 'compass-colors'
   gem 'compass-rails'
   gem 'sprockets-rails', :require => 'sprockets/railtie'
+  gem 'sprockets-es6'
+  gem 'browserify-rails'
```
Comment (schultyy) [inline Gemfile]: can you pin the version numbers here for `sprockets-es6` and `browserify-rails`?

**Comment by lennerd** [comment]
@schultyy Thanks for the feedback! Squased all commits and added changes based on feedback.

**Comment by schultyy** [comment]
LGTM

**Pull request merged**